patrol/patrol.go

```diff
@@ -37,8 +40,8 @@ type BombSquadMetricConfig struct {
 }
 
 func (p *Patrol) Run() {
-	ticker := time.NewTicker(time.Duration(p.Interval) * time.Second)
-	for _ = range ticker.C {
+	ticker := time.NewTicker(p.Interval)
```

Comment:
The recast was definitely superfluous, but I think the `* time.Second` is still needed, yea? I don't see anything in the `time.NewTicker` docs that indicates it defaults to seconds. Based on the [time.Duration](https://golang.org/pkg/time/#Duration) docs, I think this will default Patrol to tick every 5 nanoseconds.

Reply:
Right, my bad.

Check the latest commit, what we have is a duration, it should be in the seconds level (when dealing with humans) which is why what I did was pushing it to the edge of the application (I actually forgot to do it initially, so it's [here](https://github.com/Fresh-Tracks/bomb-squad/pull/19/commits/f879dd8a40a0b6c8871432ff639802ebf89e5e34)).

By defining the _seconds_ in the master application, we can set a way lower interval in testing so we don't have to wait 15 seconds to get a reading, but rather 300ms tops.

Reply:
:+1: